docs/content/en/docs/pipeline-stages/builders/docker.md

```diff
@@ -94,6 +116,33 @@ Docker image `gcr.io/k8s-skaffold/example` with Kaniko:
 
 {{% readfile file="samples/builders/kaniko.yaml" %}}
 
+### Configure Kaniko Volume Mounts
+
+You might need to configure volume mounts for a Kaniko pod either to 
+1. Mount a secret or
+2. Set up a [persistent cache](https://github.com/GoogleContainerTools/kaniko/blob/master/examples/kaniko-test.yaml#L27).
+
+To set up volume mounts, configure the `volumeMount` key in `cluster` section like this
+```yaml
+build:
+  artifacts:
+  - image: getting-started
+    kaniko:
+      cache:
+        repo: getting-started
+      volumeMounts:
+      - name: kaniko-cache
+        mountpath: /cache
+        readonly: true
+cluster:
+  volumes:
+  - name: kaniko-cache
+    persistentvolumevlaim:
```

Comment:
```suggestion
    persistentvolumeclaim:
```

Reply:
I think we need to retrieve this from a file so that we can do some validation as I think this example need another level of indirection for `volumesource` and should actually be:

```
    volumes:
      - name: kaniko-cache
        volumesource:
          persistentvolumeclaim: 
            claimname: kaniko-cache-claim
```